.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+### Summary of Changes
+
+Your summary goes here!
+
+### PR Checklist
+
+Make sure the following checklist items are done before merging this PR.
+
+- [ ] Update `CHANGELOG.md` in the `## Unreleased` section ([instructions here])
```

Comment:
I _think_ we can enforce that all checklist items are checked before merging a PR, but I'll have to verify that once we've meverged.